Menu/PageTreeMenuBuilder.php

```diff
@@ -72,6 +72,12 @@ private function appendNodes (ItemInterface $parent, array $nodes, array $option
     {
         foreach ($nodes as $node)
         {
+            // Don't render nodes that the User is not supposed/eligible to see
```

Comment:
Then how should we tackle the problem with rendering the entire PageTree bundle with "display: hidden;" inside the DOM, which might reveal data that isn't supposed to be there?